### PR TITLE
split events into spearate package

### PR DESCRIPTION
**Pull request by tomerd**
motivation: the runtime library has a stabel API while the events are still moving target, so in order to provide a 1.0 stable version we should separate them out

changes:
* remove Events module
* update readme
* update Samples

### CODE REVIEW

File: Examples/LambdaFunctions/Package.swift [hunk 1]
```diff
@@ -24,6 +24,9 @@ let package = Package(
         // in real-world projects this would say
         // .package(url: "https://github.com/swift-server/swift-aws-lambda-runtime.git", from: "1.0.0")
         .package(name: "swift-aws-lambda-runtime", path: "../.."),
+        // this is the dependency on the swift-aws-lambda-events library
+        // FIXME: update to tag when available
```
Comment (tomerd) [inline Examples/LambdaFunctions/Package.swift:28]: 👀 we need to first tag https://github.com/swift-server/swift-aws-lambda-events

**Comment by tomerd** [comment]
cc @bmoffatt @kneekey23

**Review by fabianfett — APPROVED** [review]
Looks good to me. However we should have an "integration" test target in AWSLambdaEvents that ensures that the examples in the runtime still compile after a change in events.

**Comment by tomerd** [comment]
> Looks good to me. However we should have an "integration" test target in AWSLambdaEvents that ensures that the examples in the runtime still compile after a change in events.

@fabianfett you mean in the new repo?

**Comment by fabianfett** [comment]
> @fabianfett you mean in the new "events" repo, so that changes there dont break the runtime? could they?

The changes in there can't break the runtime, but the samples in the runtime repository, which would block CI.

**Comment by tomerd** [comment]
> The changes in there can't break the runtime, but the samples in the runtime repository, which would block CI.

it may ben easier (from a CI point of view) to add Examples there as well. WDYT?

**Comment by tomerd** [comment]
https://github.com/swift-server/swift-aws-lambda-events/pull/1

**Comment by fabianfett** [comment]
@tomerd Awesome! Now we should remove the `APIGateway` example from the code here. And maybe add to the soundness script that we never import `AWSLambdaEvents` except in markdown files :)

**Comment by tomerd** [comment]
> tomerd Awesome! Now we should remove the APIGateway example from the code here. All other examples don't depend on AWSLambdaEvents. Maybe we should add to the soundness script that we never import AWSLambdaEvents except in markdown files :)

done

**Comment by fabianfett** [comment]
Failing nightly-main CI because of known issue in SwiftNIO: https://github.com/apple/swift-nio/pull/1943 Not a blocker! Merging!

**Pull request merged**